### DISCUSSION TITLE
Is there any way to get minisectors time data?

### DISCUSSION THREAD

**Discussion by Pablomgom** [theOehrly/Fast-F1#554] [Q&A]
In Multiviewer, we can see how they are taking the minisectors time data, but I don't know how are they getting this values
![1](https://github.com/theOehrly/Fast-F1/assets/106599740/27c69b56-b64b-4448-814f-691ef32878dc)

**Comment by theOehrly** [comment]
This is not implemented in FastF1 because I don't think that the data is very useful for data analysis. The problem is that there is no actual "timing" data for those mini sectors. There is just "slower", "faster" and "fastest overall" so basically the three colours yellow, green and pink (blue is in pit). But the data doesn't tell you how much slower or how much faster than someone else. That means the information that you can extract from that is very limited. For example, while you are able to tell who is fastest in a mini sector, you are already unable to tell who is second fastest in the same mini sector.
